<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountsActionMenu</class>
 <widget class="QDialog" name="AccountsActionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="btDelete">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/delete.png</normaloff>
         <disabledoff>:/icons/delete-disbl.png</disabledoff>
         <disabledon>:/icons/delete-disbl.png</disabledon>:/icons/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btAdvSettings">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/edit.png</normaloff>
         <disabledoff>:/icons/edit-disbl.png</disabledoff>:/icons/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/add.png</normaloff>
         <disabledoff>:/icons/add-disbl.png</disabledoff>:/icons/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="taki.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btDelete</sender>
   <signal>released()</signal>
   <receiver>AccountsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAdvSettings</sender>
   <signal>released()</signal>
   <receiver>AccountsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAdd</sender>
   <signal>released()</signal>
   <receiver>AccountsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
